test/plugin.test.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-env jest */
+const squid = require('flying-squid')
+const settings = require('../config/default-settings')
+const path = require('path')
+const fs = require('fs')
+
+describe('test import third-party plugins', () => {
+  const pluginPath = path.resolve(__dirname, '../', 'src', 'plugins', 'example')
+  const filePath = path.resolve(__dirname, '../', 'src', 'plugins', 'example', 'index.js')
+  beforeAll(() => {
+    if (!fs.existsSync(pluginPath)) {
+      fs.mkdirSync(pluginPath)
```

Comment:
Why are you writing the file here if you also commit it ?

Reply:
After thinking twice, I think this unit test part is unuseful, so I removed it